### PR TITLE
If Typoscript is 1 then PHP is true

### PR DESCRIPTION
**Pull request by franzholz**

### CODE REVIEW

**Review by brotkrueml — COMMENTED** [review]
Actually, I suggest always to use "0" and "1" in TypoScript/TSconfig examples (also default values). "false" is wrong in this context as this evaluates to true.

So, my suggestion: switch from "true" to "1" and "false" to "0".

**Comment by franzholz** [comment]
This means that TypoScript 1 is on the PHP side true. So this should work as it is in my patch.

**Comment by brotkrueml** [comment]
> This means that TypoScript 1 is on the PHP side true. So this should work as it is in my patch.

Not really, you use this text:

    :php:`true` or :typoscript:`1`

This is misleading, as the (unexperienced) user may think, using "true" or "false" is okay (PHP is only shown when hovering over the "true"). My suggestion is, to remove true/false altogether and just use "0" and "1".

File: Documentation/Configuration/TSconfig.rst [hunk 1]
```diff
@@ -30,11 +30,11 @@ User TSconfig
     :type: bool
     :Default: false
```
Comment (brotkrueml) [inline Documentation/Configuration/TSconfig.rst:31]:
```suggestion
    :Default: 0
```

**Review by brotkrueml — APPROVED** [review]
Fine for me.

**Review by linawolf — APPROVED** [review]

**Pull request merged**